build_all/base-libraries/AzureIoTUtility/src/esp8266/sample_init.cpp

```diff
@@ -58,6 +60,10 @@ void esp8266_sample_init(const char* ssid, const char* password)
     initSerial();
     initWifi(ssid, password);
     initTime();
+    /* Uncomment wdt_disable() and comment out next line when gdb debugging. 
+    Note: use of system_get_free_heap_size() function may also help in tracking available memory. */
+    //wdt_disable(); 
```

Comment:
>//wdt_disable();  [](start = 4, length = 17)

remove dead code?

Reply:
This was left in to assist in debugging if a user desires, kind of a helpful comment.